app/main.py

```diff
@@ -1,3 +1,29 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
```

Comment:
You need to store some logic to this function. You create objects so can store them to variables and call methods of classes and put them as an parameters of those methods. The other developers should open this main module and understand the main logic of your project